ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -369,13 +369,27 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
   async saveInstitution() {
     const {institution, institutionMode} = this.state;
     const rtConfig: InstitutionTierConfig = getRegisteredTierConfig(institution);
+    const ctConfig: InstitutionTierConfig = getControlledTierConfig(institution);
     if (institution && rtConfig) {
       if (rtConfig.membershipRequirement === InstitutionMembershipRequirement.DOMAINS) {
         rtConfig.emailAddresses = [];
       } else if (rtConfig.membershipRequirement === InstitutionMembershipRequirement.ADDRESSES) {
         rtConfig.emailDomains = [];
       }
     }
+    if (institution && ctConfig) {
+      if (ctConfig.membershipRequirement === InstitutionMembershipRequirement.DOMAINS) {
+        ctConfig.emailAddresses = [];
+      } else if (ctConfig.membershipRequirement === InstitutionMembershipRequirement.ADDRESSES) {
+        ctConfig.emailDomains = [];
+      }
+    }
+    if (ctConfig.membershipRequirement === InstitutionMembershipRequirement.NOACCESS) {
```

Comment:
is this necessary if we are checking this in the API code?  I'd prefer not to special-case this if we don't have to.

Reply:
I think that is necessary and safer way to control it. Not 100% confident if API capture all tier not exist cases. But setting it here is more clear: If user choose no_access, just delete the entier tier config